package.json

```diff
@@ -344,15 +344,17 @@
     "@brave/react-virtualized-auto-sizer": "^1.0.4",
     "@glif/filecoin-wallet-provider": "2.0.0-beta.12",
     "@ledgerhq/hw-app-eth": "6.24.1",
+    "@ledgerhq/hw-app-solana": "^6.27.1",
```

Comment:
please pin the exact version

Reply:
also have we considered doing the ledger integration in a chrome-untrusted frame like we do with trezor?

Reply:
@nvonpentz that would be good to keep in line with our principle that non-trivial new dependencies should be isolated from the main `chrome://wallet` context where possible. i'm guessing that here you would just put the entire hardware wallet integration webui in a chrome-untrusted frame, and hopefully the other ledgerhq deps could be also moved into this frame: https://github.com/brave/brave-core/pull/13734/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R346-R348

you can see the trezor example here btw: https://github.com/brave/brave-core/pull/10681

Reply:
@diracdeltas Thanks, I replied in the security [issue](https://github.com/brave/security/issues/919#issuecomment-1158884294).

Reply:
We will follow up and refactor all ledger integrations to use chrome-untrusted https://github.com/brave/brave-browser/issues/23655